resources/views/checkout/pages/credentials.blade.php

```diff
@@ -0,0 +1,40 @@
+@extends('rapidez::layouts.app')
+
+@section('title', __('Checkout'))
+
+@section('robots', 'NOINDEX,NOFOLLOW')
+
+@section('content')
+    <div class="container">
+        <x-rapidez-ct::layout class="mt-8 sm:mt-14">
+            <x-rapidez-ct::title-progress-bar :href="route('cart')" :$checkoutSteps :$currentStep :$currentStepKey>
+                @lang('Credentials')
+            </x-rapidez-ct::title-progress-bar>
+
+            <form
+                class="contents"
+                v-on:submit.prevent="(e) => {
```

Comment:
Form with contents, i know this fixes the styling but this isn't really the way to go in my opinion. Ill put it on my list to look at this and tag you whenever i have a counter suggestion!